src/store/store.ts

```diff
@@ -1294,25 +1295,25 @@ export const useStore = defineStore('store', {
     setEntityType (entityType: CorpTypeCd) {
       this.stateModel.tombstone.entityType = entityType
     },
-    setBusinessId (businessId) {
+    setBusinessId (businessId: string) {
       this.stateModel.tombstone.businessId = businessId
     },
-    setIsSaving (isSaving) {
+    setIsSaving (isSaving: boolean) {
       this.stateModel.tombstone.isSaving = isSaving
     },
-    setIsSavingResuming (isSavingResuming) {
+    setIsSavingResuming (isSavingResuming: boolean) {
       this.stateModel.tombstone.isSavingResuming = isSavingResuming
     },
-    setIsFilingPaying (isFilingPaying) {
+    setIsFilingPaying (isFilingPaying: boolean) {
       this.stateModel.tombstone.isFilingPaying = isFilingPaying
     },
-    setKeycloakRoles (keycloakRoles) {
+    setKeycloakRoles (keycloakRoles: string[]) {
       this.stateModel.tombstone.keycloakRoles = keycloakRoles
     },
-    setUserInfo (userInfo) {
+    setUserInfo (userInfo: any) {
       this.stateModel.tombstone.userInfo = userInfo
     },
-    setOrgInfo (orgInfo) {
+    setOrgInfo (orgInfo: any) {
```

Comment:
Should I create a new interface for orgInfo (and userInfo) to avoid using `any`?

Reply:
Unless other reviewers disagree, I believe `any` here should be OK.

Reply:
Unless other reviewers disagree, I believe any here should be OK.

Reply:
This and userInfo above are declared as `any` in the TombStoneIF, so this is OK with me.

Ideally, we'd have an interface for both of these, but that's outside the scope of this ticket so don't worry about it.

Reply:
Leo, you can resolve this conversation (unless you want it visible still).